<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="RuntimeBinderException.xml" source-language="en-US" target-language="it-IT">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac57e823942ca7a2763556fa1c479a6d93554498144.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">7e823942ca7a2763556fa1c479a6d93554498144</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException">
          <source>Represents an error that occurs when a dynamic bind in the C# runtime binder is processed.</source>
          <target state="translated">Rappresenta un errore che si verifica quando viene elaborata un'associazione dinamica nel binder di runtime in C#.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException">
          <source>Exceptions of this kind differ from <ph id="ph1">&lt;xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException&gt;</ph> in that <ph id="ph2">&lt;xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&gt;</ph> represents a failure to bind in the sense of a usual compiler error, whereas <ph id="ph3">&lt;xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException&gt;</ph> represents a malfunctioning of the runtime binder itself.</source>
          <target state="translated">Le eccezioni di questo tipo differiscono da <ph id="ph1">&lt;xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException&gt;</ph> in <ph id="ph2">&lt;xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&gt;</ph> rappresenta un errore di associazione, di un errore del compilatore, mentre <ph id="ph3">&lt;xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException&gt;</ph> rappresenta un malfunzionamento del binder di runtime se stesso.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" uid="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" /&gt;</ph> class.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" /&gt;</ph> class.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> property of the new instance to a system-supplied message that describes the error, such as "DefaultMessageDisplayedByParameterlessCtorWriterMustSupply" This message takes into account the current system culture.</source>
          <target state="translated">Questo costruttore inizializza la <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> proprietà della nuova istanza di un messaggio fornito dal sistema che descrive l'errore, ad esempio "DefaultMessageDisplayedByParameterlessCtorWriterMustSupply" questo messaggio prende in considerazione il sistema corrente impostazioni cultura.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.InsufficientMemoryException&gt;</ph>.</source>
          <target state="translated">Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un'istanza di <ph id="ph1">&lt;xref:System.InsufficientMemoryException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor">
          <source>Property</source>
          <target state="translated">Proprietà</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor">
          <source>Value</source>
          <target state="translated">Valore</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor">
          <source><ph id="ph1">`null`</ph>.</source>
          <target state="translated"><ph id="ph1">`null`</ph>.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor">
          <source>The localized error message string.</source>
          <target state="translated">Stringa del messaggio di errore localizzato.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)">
          <source>The message that describes the exception.</source>
          <target state="translated">Messaggio in cui viene descritta l'eccezione.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Il chiamante del costruttore deve garantire che la stringa sia stata localizzata in base alle impostazioni cultura correnti del sistema.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" /&gt;</ph> class that has a specified error message.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" /&gt;</ph> che include un messaggio di errore specificato.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&gt;</ph>.</source>
          <target state="translated">Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un'istanza di <ph id="ph1">&lt;xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)">
          <source>Property</source>
          <target state="translated">Proprietà</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)">
          <source>Value</source>
          <target state="translated">Valore</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)">
          <source><ph id="ph1">`null`</ph>.</source>
          <target state="translated"><ph id="ph1">`null`</ph>.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)">
          <source>The error message string specified in <ph id="ph1">`message`</ph>.</source>
          <target state="translated">Stringa del messaggio di errore specificata in <ph id="ph1">`message`</ph>.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that holds the serialized object data about the exception being thrown..</source>
          <target state="translated">Oggetto nel quale sono contenuti i dati di oggetto serializzati relativi all'eccezione generata.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The contextual information about the source or destination.</source>
          <target state="translated">Informazioni contestuali sull'origine o sulla destinazione.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" /&gt;</ph> class that has serialized data.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" /&gt;</ph> che include dati serializzati.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">Questo costruttore viene chiamato durante la deserializzazione per ricostituire l'oggetto eccezione trasmesso su un flusso.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Messaggio di errore in cui viene indicato il motivo dell'eccezione.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception, or a null reference if no inner exception is specified.</source>
          <target state="translated">Eccezione che ha provocato l'eccezione corrente o riferimento null se non è stata specificata alcuna eccezione interna.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" /&gt;</ph> class that has a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException" /&gt;</ph> che include un messaggio di errore specificato e un riferimento all'eccezione interna che ha generato l'eccezione.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Un'eccezione generata come risultato diretto di un'eccezione precedente deve includere un riferimento all'eccezione precedente nella proprietà <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">La proprietà <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> restituisce lo stesso valore passato al costruttore oppure <ph id="ph2">`null`</ph> nel caso in cui la proprietà <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> non fornisca il valore di eccezione interna al costruttore.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&gt;</ph>.</source>
          <target state="translated">Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un'istanza di <ph id="ph1">&lt;xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">Proprietà</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">Valore</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)">
          <source><ph id="ph1">`null`</ph>.</source>
          <target state="translated"><ph id="ph1">`null`</ph>.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT" uid="M:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)">
          <source>The error message string specified in <ph id="ph1">`message`</ph>.</source>
          <target state="translated">Stringa del messaggio di errore specificata in <ph id="ph1">`message`</ph>.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>